Source/Editor/Modules/PrefabsModule.cs

```diff
@@ -37,6 +37,8 @@ public sealed class PrefabsModule : EditorModule
         /// </summary>
         public event Action<Prefab, Actor> PrefabApplied;
 
+        private Actor _actor;
```

Comment:
Could you add a comment that it's locally cached actor ref for prefab creation? And maybe rename it to something more self-explanatory.